policyengine_us/parameters/gov/states/nh/tax/income/rate.yaml

```diff
@@ -1,6 +1,6 @@
-description: New Hampshire taxes bonds, notes, and money at interest at this rate.
+description: New Hampshire taxes interest and dividends at this rate.
 values:
-  2022-01-01: 0.05
+  2021-01-01: 0.05
   2023-01-01: 0.04
   2024-01-01: 0.03
   2025-01-01: 0.02
```

Comment:
While you're here could you add `2027-01-01: 0` per the section being repealed effective 2027?

Reply:
Done